c++/include/orc/Reader.hh

```diff
@@ -288,6 +288,18 @@ namespace orc {
     virtual uint64_t getCompressionSize() const = 0;
 
     /**
+     * Get ID of writer that generated the file.
+     * @return UNKNOWN_WRITER if the writer ID is undefined
+     */
+    virtual WriterId getWriterId() const = 0;
+
+    /**
+     * Get the writer id value when getWriterId() returns an unknown writer.
+     * @return the integer value of the writer ID.
+     */
+    virtual int getUnknownWriterIdValue() const = 0;
```

Comment:
Should this be `getWriteIdValue`? I think the implementation returns both known and unknown ids?

Reply:
Sounds good. I will update it. Thanks.